src/browser/modules/Carousel/styled.tsx

```diff
@@ -302,3 +304,120 @@ export const StyledSlide = styled.div`
     background-color: ${props => props.theme.teaserCardBackground} !important;
   }
 `
+
+export const StyledSidebarSlide = styled.div.attrs({
+  className: 'slide' /* added to get styling from less.css */
+})`
+  color: ${dark.primaryText};
+  & p.lead,
+  .title,
+  .subtitle,
+  .content > p,
+  .table-help {
+    color: ${dark.primaryText} !important;
+    line-height: 1.3;
+
+    th {
+      padding-right: 20px;
+      text-align: left;
+    }
+
+    &--header {
+      th {
+        border-bottom: ${dark.topicBorder};
+        font-size: 2rem;
+        padding: 15px 0 0 0;
+      }
+
+      &:first-child {
+        th {
+          padding-top: 0;
+        }
+      }
+    }
+
+    &--subheader {
+      th {
+        border-bottom: ${dark.topicBorder};
+        font-size: 1.5rem;
+        padding: 10px 0 0 0;
+      }
+
+      &:first-child {
+        th {
+          padding-top: 0;
+        }
+      }
+    }
+
+    &--commands {
+      margin-top: 2rem;
+
+      td {
+        padding: 3px 10px 3px 0;
+      }
+    }
+
+    &--keys {
+      th {
+        border-bottom: ${dark.topicBorder};
+      }
+      td {
+        padding: 3px 10px 3px 0;
+      }
+    }
+  }
+  & a {
+    color: ${dark.link};
+    text-decoration: ${props =>
+      props.theme.name === 'dark' ? 'underline' : 'none'};
+  }
+  & kbd {
+    color: ${dark.primaryBackground} !important; /* inverted */
+    background-color: ${dark.primaryText} !important;
+  }
+  & .content > pre {
+    background-color: ${dark.secondaryBackground};
+    color: ${dark.preText};
+  }
+  & pre.runnable {
+    background-color: ${dark.preBackground};
+    color: ${dark.preText};
+  }
+  & pre.content {
+    background-color: ${dark.secondaryBackground};
+    color: ${dark.preText};
+  }
+  & a[help-topic],
```

Comment:
is it intentional that the first one had `&` prefix but the others don't? Also it feels like it could be better to just use a classname to simplify this kind of styling...